pkg/vpnc-scripts-tarball

```diff
@@ -0,0 +1,16 @@
+[mirrors]
+https://github.com/alonbl/vpnc-scripts/archive/master.zip
+
+[vars]
+filesize=21574
+sha512=e2c7d3543540b2a22fba9e1eda09cda1dda79c540e7edb7c5c164e4228867b24dafcdf1b070bbd49f618ac6dbbbd1e3f4e1be64e727065ca32f55602fdb0718d
+pkgver=1
+tarball="vpnc-scripts.zip"
+
+[build]
```

Comment:
a "tarball-provider" package shouldn't have a [build] section - what you do here should be done from the openconnect package